baselines/aad.md

```diff
@@ -640,109 +617,128 @@ User passwords SHALL NOT expire.
+### Resources
 
-2.  Set the **Cloud apps or actions** to include **All cloud apps.**
+- [Best practices for Azure AD roles (Limit number of Global
+  Administrators to less than 5)](https://docs.microsoft.com/en-us/azure/active-directory/roles/best-practices#5-limit-the-number-of-global-administrators-to-less-than-5)
 
-3.  Set the **Access Controls -**\> **Session** -\> **Persistent browser
-    session** to **Never persistent.**
+- [About admin roles](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/about-admin-roles?view=o365-worldwide)
 
-## 11. Global Administrators
+- [Securing privileged access for hybrid and cloud deployments in Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/roles/security-planning#ensure-separate-user-accounts-and-mail-forwarding-for-global-administrator-accounts)
 
-Global Administrator is the highest privileged role in Azure AD because
-it provides unfettered access to the tenant. Therefore, if a user’s
-credential with these permissions were to be compromised, it would
-present grave risks to the security of the tenant. Limit the number of
-users that are assigned the role of Global Administrator. Assign users
-to finer-grained administrative roles that they need to perform their
-duties instead of being assigned the Global Administrator role.
+- [Five steps to securing your identity infrastructure](https://docs.microsoft.com/en-us/azure/security/fundamentals/steps-secure-identity)
 
-### Policies
-#### MS.AAD.11.1v1
-A minimum of two users and a maximum of four users SHALL be provisioned with the Global Administrator role.
-- _Rationale:_ TODO
-- _Last modified:_ June 2023
+- [M-22-09 Federal Zero Trust Strategy](https://www.whitehouse.gov/wp-content/uploads/2022/01/M-22-09.pdf)
 
-### Resources
+- [Assign Azure AD roles in Privileged Identity Management](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-how-to-add-role-to-user)
 
-- [Best practices for Azure AD roles (Limit number of Global
-  Administrators to less than 5)](https://docs.microsoft.com/en-us/azure/active-directory/roles/best-practices#5-limit-the-number-of-global-administrators-to-less-than-5)
+- [Approve or deny requests for Azure AD roles in Privileged Identity Management](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/azure-ad-pim-approval-workflow)
 
-- [About admin roles](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/about-admin-roles?view=o365-worldwide)
+- [Assign Azure AD roles in Privileged Identity Management](https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-how-to-add-role-to-user)
 
 ### License Requirements
 
-- N/A
-
-### Implementation
+- Use of an Azure AD PIM or an equivalent third-party PAM service.
 
-**Policy bullet \#1:**
+- Azure AD PIM requires an AAD P2 license
 
+### Implementation
+#### Policy MS.AAD.11.1v1
```

Comment:
Other implementation sections using asterisk *.  Be consistent?

Reply:
Modified all implementation sections to use the correct heading tag and ensure it aligns with the standard baseline format we agreed on.

Reply:
I think this breaks the current parsing of the baseline import.  It will be an easy fix but I'll opened an issue. #405